.github/workflows/perf-models.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
       - uses: ./.github/actions/install-python-deps
       - name: Run performance regressions
         id: performance_tests
-        timeout-minutes: 30
+        timeout-minutes: 60
```

Comment:
Is there a reason for increasing timeouts?

Reply:
I want to give wiggle room if something hangs so it can reset and hopefully run through the whole suite still. I've made it 40 mins now, hopefully that's more reasonable?